lib/src/mjpeg.dart

```diff
@@ -132,71 +133,72 @@ class _StreamManager {
     _httpClient.close();
   }
 
-  Future<void> _sendImage(BuildContext context, ValueNotifier<MemoryImage?> image, ValueNotifier<dynamic> errorState, List<int> chunks) async {
+void _sendImage(BuildContext context, ValueNotifier<MemoryImage?> image, ValueNotifier<dynamic> errorState, List<int> chunks)  {
     final imageMemory = MemoryImage(Uint8List.fromList(chunks));
     try {
-      await precacheImage(imageMemory, context, onError: (err, trace) {
+      precacheImage(imageMemory, context, onError: (err, trace) {
```

Comment:
I suppose by removing that the precacheImage can be totally removed ^^ as you push the image before it's even pre cached :D